lib/jekyll/commands/new.rb

```diff
@@ -21,8 +22,11 @@ def init_with_program(prog)
         def process(args, options = {})
           raise ArgumentError, "You must specify a path." if args.empty?
 
+          new_blog_title = sanitized args
```

Comment:
the reason i delegated extraction to a standalone method is because it got too complicated (for me) further ahead.

by the time the following queued up for processing, `new_blog_title` was already defined.
```ruby
def create_site(new_blog_title, new_blog_path)
  create_sample_files new_blog_title, new_blog_path
  [...]
end
```

But if were to have `new_blog_title` itself be a method, then (with all due respect), things go beyond my newbie-skill-level. Sorry.
The max I can do to this (as of this writing) is to rename `sanitized` to `extract_title`